D11 Availability on Pantheon (#9172)
pantheon-systems/documentation: source/content/supported-drupal.md

| Drupal Version | Available | Recommended | Supported |
| ----------- | :---------: | :---------: | :---------: |

> | 11          | ⚠️ <Popover title="Drupal 11 Availability" content="See the <a href='#drupal-11-on-pantheon'>following section</a> for how to run Drupal 11 on Pantheon today." />        | <span style="color:green">✔</span>           | <span style="color:green">✔</span>

Reviewer: I think "set up" is maybe more accurate than "run". Minor nit, though.
```suggestion
| 11          | ⚠️ <Popover title="Drupal 11 Availability" content="See the <a href='#drupal-11-on-pantheon'>following section</a> for how to set up Drupal 11 on Pantheon today." />        | <span style="color:green">✔</span>           | <span style="color:green">✔</span>
```

| 10          | <span style="color:green">✔</span>         | <span style="color:green">✔</span>           | <span style="color:green">✔</span>          |
| 9           | <span style="color:green">✔</span>          | <span style="color:green">✔</span>           | <span style="color:green">✔</span>         |
| 8           | <span style="color:green">✔</span>          | ❌           | <span style="color:green">✔</span>         |
| 7           | <span style="color:green">✔</span>         | ❌           | <span style="color:green">✔</span>          |
| 6           | ❌          | ❌           | ❌          |

## Drupal 11 on Pantheon

> Drupal 11 is not yet an option when creating a new site and there is no current ETA for when it will be added. However, Drupal 11 has been tested and is reported to work on Pantheon today.

Reviewer: I don't like calling out that we don't have an ETA. It feels better to just say "it's coming", which is still true. 
Updated the language from "reported to work" to something that sounds more confident.

```suggestion
Drupal 11 is not yet an option when creating a new site. It will be added to the Pantheon dashboard in the coming weeks. However, Drupal 11 has been tested and works on Pantheon environments.
```

> To run Drupal 11 on the platform, you must [create a new Drupal 10 site](/add-site-dashboard) then [upgrade to Drupal 11 via composer](https://www.drupal.org/docs/upgrading-drupal/upgrading-from-drupal-8-or-later/how-to-upgrade-from-drupal-10-to-drupal-11).

Reviewer: ```suggestion
To install Drupal 11 on your site, you must [create a new Drupal 10 site](/add-site-dashboard) then [upgrade to Drupal 11 via composer](https://www.drupal.org/docs/upgrading-drupal/upgrading-from-drupal-8-or-later/how-to-upgrade-from-drupal-10-to-drupal-11).
```

Reviewer: There should also be some mention here that if you _already have_ a D10 site, you can just bump the version to get 11. This makes it sound like _the only way_ to get D11 is creating a brand new site.
